example_dags/transfer_comparison_with_traditional_transfer_operator.py

```diff
@@ -0,0 +1,69 @@
+import os
+from datetime import datetime
+
+from airflow import DAG
+from airflow.providers.google.cloud.transfers.s3_to_gcs import S3ToGCSOperator
+from airflow.providers.snowflake.transfers.s3_to_snowflake import S3ToSnowflakeOperator
+
+from universal_transfer_operator.constants import FileType
+from universal_transfer_operator.datasets.file.base import File
+from universal_transfer_operator.datasets.table import Table
+from universal_transfer_operator.universal_transfer_operator import UniversalTransferOperator
+
+s3_bucket = os.getenv("S3_BUCKET", "s3://astro-sdk-test")
+gcs_bucket = os.getenv("GCS_BUCKET", "gs://uto-test")
+
+
+with DAG(
+    "transfer_comparison_with_tradition_transfer_operator",
+    schedule_interval=None,
+    start_date=datetime(2022, 1, 1),
+    catchup=False,
+) as dag:
+    # transfer file from S3 to GCS using universal transfer operator
```

Comment:
it would be nice to have a `howto` tag here and refer the block in the docs

Reply:
Added `howto` here: https://github.com/astronomer/apache-airflow-provider-transfers/pull/37/commits/2a40ad87609174a686debb80e2907b3c0268737a